src/components/pages/main/MainPage.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useCallback, useEffect, useState } from "react";
+import BooksList from "./components/booksList/BooksList";
+import { IBook } from "../../../interfaces";
+import { apiService } from "../../../services/ApiService";
+import SearchBar from "./components/searchBar/SearchBar";
+import { FetchStatusEnum } from "../../../enums/FetchStatusEnum";
+import { Outlet, useSearchParams } from "react-router-dom";
+import "./MainPage.css";
+import useLocalStorage from "../../../utils/useLocalStorage";
+import { ThemeContext } from "../../../ThemeContext";
+
+interface MainPageProps {}
+
+const MainPage: React.FC<MainPageProps> = () => {
```

Comment:
```suggestion


const MainPage = () => {
```
если пропсов нет, то можешь вообще ничего не писать